corehq/apps/user_importer/tasks.py

```diff
@@ -102,6 +103,9 @@ def import_users(domain, user_specs, group_specs, upload_user_id, upload_record_
     def _update_progress(value, start=0):
         DownloadBase.set_progress(task, start + value, total)
 
+    from corehq.apps.users.views.mobile.users import MobileWorkerListView
```

Comment:
Up to you, but I'd consider whether `check_and_send_limit_email` belongs inside a view module. I don't think it's functionality inherently fits with a view. It feels a bit odd to see this importer task forced to import from a view module to get this functionality

Reply:
Do you mind explaining why this is odd? I'm a little confused by what you mean by its "inherent functionality", though I do see that it would also fit neatly as a separate task in tasks.py. But then the view function would have to import it as well, which I guess could avoid the circular import issue it's having now.

Reply:
I think it was explained to me that in architectures like Model-View-Controller (MVC) or N-Tier, each tier only knows about the tiers beneath it. So a View can know about and use Model code, but not the other way around (you can see how this avoids circular dependencies). I've traditionally viewed View code as the end tier, where no code knows about it except other View code. So that's why its weird to see behavior that lower levels might use, put into the highest level.

Reply:
that's a helpful way of looking at it, thanks! Moved to models.py